### PR TITLE
Add last modified author and other minor document changes

### PR DESCRIPTION
**Pull request by shoichiaizawa**
### TL;DR

- Add last modified author feature
- Change `README.md`

### This PR includes the following changes:

- feat: add last modified author
  - The majority of this code is reused from the modified timestamp code
  - This feature is inspired by the Atom's `file-header` plugin (https://github.com/guiguan/file-header/)
  - The `Last Modified:` label has been changed to `Last Modified Date:` according to this change
- docs: update README.md for last modified author
  - Update README.md according to the new last modified author feature
  
  NB: I am not a native speaker of English; please review what I have changed.

- refactor: include tmux filetype to existing elseif block

  In a previous commit, new filetypes: ruby and elixir were added by
  @iomonad. Since the comment character for these two languages is a hash
  (#), which is the same for tmux, the conditional statement for detecting
  tmux filetype has been included to the `elseif` block for ruby and
  elixir.

- docs: change markdown headings

    At the top of README.md, there was a line goes like this: `#vim-header`;
    using the `===` Markdown syntax to make sure that this line will be
    rendered as a H1 heading.

    The similar change applies for `###Thanks to Contributors` line,
    changing it to be rendered as a H3 heading.

    This could be a matter of style preference but all other H1 headings
    have been converted to H2 headings.

- docs: fix a minor Markdown rendering mistake

    In Markdown (at least this is observed for GFM), a pair of inner
    backticks are recognised as normal strings inside a pair of outer
    backticks if it is written like this: `<``>`. Insert a space character
    like this: `<` `>` so that it will be render properly.

### Notes

There are a few minor things I was thinking regarding the last modified author feature:

  - The newly modified `Last Modified Date:` label and the newly added `Last Modified By:` label are both capitalised in every letter. I was thinking of changing these to `Last modified date:` and `Last modified by:`. I wanted to hear your preference.
  - The order of headers is currently as follows:

    ```
    # File: filename.extension
    # Author: Firstname Lastname <username@example.com>
    # Date: d.m.Y
    # Last Modified Date: d.m.Y
    # Last Modified By: Firstname Lastname <username@example.com>
    ```
    I was thinking that what's the most natural order for this but for now I located the `Last Modified By:` line at the bottom so that `Date:` and `Last Modified Date:` are easily comparable. Please change as you feel natural.
  - I was thinking of including a username (perhaps for GitHub) somewhere but did not do it in the end

---

### Finally

Please have a look if this is feature is suitable for this plugin (and double-check if this works properly).

I have not changed `CHANGELOG.md` as I don't know about how to write properly (I am a OSS beginner); please let me not if doing so is a part of collaborator's responsibility.

NB: When I committed for these changes, I used AngularJS styled (ish) commit message, whose subject line exceed little more than 50 characters, which is not recommended by a traditional Git style guide.

### CODE REVIEW

**Comment by alpertuna** [comment]
Hi @shoichiaizawa,
It looks a great job here.

"Last Modified By" feature is a good idea. And the order of fields is reasonable. Also thanks for other fixes.

About changelog, you are also able to edit it. If there is something wrong, I will already fix that. And I use this [Keep A Changelog](http://keepachangelog.com/en/1.0.0/) standard to write it.

I will try and merge it. Thanks for trying hard and detailed report. :+1:
